packages/block-library/src/cover/edit.js

```diff
@@ -355,6 +358,13 @@ function CoverEdit( {
 		isVideoBackground ||
 		( isImageBackground && ( ! hasParallax || isRepeated ) );
 
+	const imperativeFocalPointPreview = ( value ) => {
+		const [ styleOfRef, property ] = isDarkElement.current
+			? [ isDarkElement.current.style, 'objectPosition' ]
+			: [ blockProps.ref.current.style, 'backgroundPosition' ];
+		styleOfRef[ property ] = mediaPosition( value );
```

Comment:
This ternary + destructuring is really cool! Will add this little pattern to my mental list cool JS patterns :)